packages/react-native/Libraries/ReactNative/ProdAppContainer.js

```diff
@@ -0,0 +1,53 @@
+/**
+ * Copyright (c) Meta Platforms, Inc. and affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @flow strict-local
+ * @format
+ * @oncall react_native
+ */
+
+import type {Props} from './AppContainer';
+
+import View from '../Components/View/View';
+import StyleSheet from '../StyleSheet/StyleSheet';
+import {RootTagContext, createRootTag} from './RootTag';
+import * as React from 'react';
+
+const ProdAppContainer = ({
+  children,
+  fabric,
+  initialProps,
+  rootTag,
+  showArchitectureIndicator,
+  WrapperComponent,
+}: Props): React.Node => {
+  let innerView = children;
+
+  if (WrapperComponent != null) {
+    innerView = (
+      <WrapperComponent
+        initialProps={initialProps}
+        fabric={fabric === true}
+        showArchitectureIndicator={showArchitectureIndicator === true}>
+        {children}
+      </WrapperComponent>
+    );
```

Comment:
We could just return the component and avoid the `innerView` variable

Reply:
Technically, yes. I was copying previous implementation and removed everything what was dev-only. 
Also, there should be `{innerView}` as children on line 35, instead of `children`.